src/index.scss

```diff
@@ -5,7 +5,6 @@ body {
 .container {
   max-width: 1200px;
   margin: 0 auto;
-  padding: 24px;
```

Comment:
Don't remove this padding, we should ensure that the container class is reusable and provides the same spacing across the page.

Reply:
Ok, restored then.